### ISSUE TITLE
[1.10.2] GuiTeleporting crash en route to the Moon [FoamFix, Optifine, etc]

### ISSUE THREAD

**Issue by metrotyranno** [micdoodle8/Galacticraft#2769]
Was heading for the moon and was stuck on "preparing for entry", well over 5 minutes decided to kick myself and see what happened when I rejoin. 30 seconds or so it stayed on preparing for entry and then the following crash:

---- Minecraft Crash Report ----

WARNING: coremods are present:
  CoreMod (Aroma1997Core-1.9.4-1.1.1.2.jar)
  Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.6.3-anarchy.jar)
  SFRCore (SolarFluxReborn_1.10.2-2.17r.jar)
  MalisisCorePlugin (malisiscore-1.10.2-4.4.0.jar)
  EnderCorePlugin (EnderCore-1.10.2-0.4.1.66-beta.jar)
  ForgelinPlugin (Forgelin-1.4.2.jar)
  MalisisSwitchesPlugin (malisisswitches-1.10.2-3.2.0.jar)
  MicdoodlePlugin (MicdoodleCore-1.10.2-4.0.0.100.jar)
  ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.9.jar)
  Default Options (DefaultOptions_1.10.2-6.1.5.jar)
  NEICorePlugin (NotEnoughItems-1.10.2-2.1.3.220-universal.jar)
  RebornCoreASM (RebornCore-1.10.2-2.13.6.142-universal.jar)
  FMLPlugin (InventoryTweaks-1.61-58.jar)
  CorePlugin (MrTJPCore-1.10.2-2.0.0.17-universal.jar)
  BookshelfLoadingPlugin (Bookshelf-1.10.2-1.4.4.347.jar)
  IC2core (industrialcraft-2-2.6.210-ex110.jar)
  DynamicSurroundingsCore (DynamicSurroundings-1.10.2-3.4.5.0.jar)
  LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  Brandon's Core (BrandonsCore-1.10.2-2.1.7.102-universal.jar)
  ChiselCorePlugin (Chisel-MC1.10.2-0.0.10.22.jar)
  CCLCorePlugin (CodeChickenLib-1.10.2-2.5.8.255-universal.jar)
  TransformerLoader (OpenComputers-MC1.10.2-1.6.2.7.jar)
  HammerCoreCore (HammerCore_1.10.2-1.6.9.jar)
  LoadingPlugin (Quark-r1.1-70.jar)
  AppEngCore (appliedenergistics2-rv4-alpha-11.jar)
  ArsMagica2-Preloader (ArsMagica2-1.10.2-1.5.0-18.jar)
  CoFH Loading Plugin (CoFHCore-1.10.2-4.1.1.156-universal.jar)
Contact their authors BEFORE contacting forge

// Daisy, daisy...

Time: 5/28/17 9:17 PM
Description: Ticking screen

java.lang.NullPointerException: Ticking screen
	at micdoodle8.mods.galacticraft.core.client.gui.screen.GuiTeleporting.func_73876_c(GuiTeleporting.java:43)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1720)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1055)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at micdoodle8.mods.galacticraft.core.client.gui.screen.GuiTeleporting.func_73876_c(GuiTeleporting.java:43)

-- Affected screen --
Details:
	Screen name: micdoodle8.mods.galacticraft.core.client.gui.screen.GuiTeleporting
Stacktrace:
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1720)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1055)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_102, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 3479536816 bytes (3318 MB) / 7516192768 bytes (7168 MB) up to 7516192768 bytes (7168 MB)
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms7168m -Xmx7168m -XX:+UseG1GC -XX:MaxGCPauseMillis=4
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.32 Powered by Forge 12.18.3.2297 Optifine OptiFine_1.10.2_HD_U_D8 223 mods loaded, 223 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAAAA	FML{8.0.99.99} [Forge Mod Loader] (modpack.jar) 
	UCHIJAAAAAA	Forge{12.18.3.2297} [Minecraft Forge] (modpack.jar) 
	UCHIJAAAAAA	appliedenergistics2-core{rv4-alpha-11} [Applied Energistics 2 Core] (minecraft.jar) 
	UCHIJAAAAAA	Aroma1997Core{${version}} [Aroma1997Core] (Aroma1997Core-1.9.4-1.1.1.2.jar) 
	UCHIJAAAAAA	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UCHIJAAAAAA	NotEnoughItems{2.1.3.220} [Not Enough Items] (NotEnoughItems-1.10.2-2.1.3.220-universal.jar) 
	UCHIJAAAAAA	OpenComputers|Core{1.6.2.7} [OpenComputers (Core)] (minecraft.jar) 
	UCHIJAAAAAA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJAAAAAA	foamfixcore{7.7.4} [FoamFixCore] (minecraft.jar) 
	UCHIJAAAAAA	ccl-entityhook{1.0} [ccl-entityhook] (CodeChickenLib-1.10.2-2.5.8.255-universal.jar) 
	UCHIJAAAAAA	fenceoverhaul{1.2.1} [Fence Overhaul] ([1.10.x]FenceOverhaul-1.2.1.jar) 
	UCHIJAAAAAA	cofhcore{4.1.1} [CoFH Core] (CoFHCore-1.10.2-4.1.1.156-universal.jar) 
	UCHIJAAAAAA	actuallyadditions{1.10.2-r105} [Actually Additions] (ActuallyAdditions-1.10.2-r105.jar) 
	UCHIJAAAAAA	IC2{2.6.210-ex110} [IndustrialCraft 2] (industrialcraft-2-2.6.210-ex110.jar) 
	UCHIJAAAAAA	advanced_machines{60.0.4} [Advanced Machines] (Advanced Machines-60.0.4.jar) 
	UCHIJAAAAAA	advanced_solar_panels{4.0.4} [Advanced Solar Panels] (Advanced Solar Panels-4.0.4.jar) 
	UCHIJAAAAAA	appliedenergistics2{rv4-alpha-11} [Applied Energistics 2] (appliedenergistics2-rv4-alpha-11.jar) 
	UCHIJAAAAAA	bdlib{1.12.4.24} [BD Lib] (bdlib-1.12.4.24-mc1.10.2.jar) 
	UCHIJAAAAAA	ae2stuff{0.6.0.9} [AE2 Stuff] (ae2stuff-0.6.0.9-mc1.10.2.jar) 
	UCHIJAAAAAA	infinitylib{1.10.2-0.10.0} [InfinityLib] (infinitylib-0.10.0.jar) 
	UCHIJAAAAAA	agricraft{2.0.0-0.10.0-a17} [AgriCraft] (agricraft-2.0.0-0.10.0-a17.jar) 
	UCHIJAAAAAA	Baubles{1.3.11} [Baubles] (Baubles-1.10.2-1.3.11.jar) 
	UCHIJAAAAAA	Botania{r1.9-341} [Botania] (Botania r1.9-341.jar) 
	UCHIJAAAAAA	AkashicTome{1.0-5} [Akashic Tome] (AkashicTome-1.0-5.jar) 
	UCHIJAAAAAA	animania{1.0.4.7} [Animania] (animania-1.10.2-1.0.4.7.jar) 
	UCHIJAAAAAA	JEI{3.14.7.419} [Just Enough Items] (jei_1.10.2-3.14.7.419.jar) 
	UCHIJAAAAAA	appleskin{1.0.5} [AppleSkin] (AppleSkin-mc1.10.2-1.0.5.jar) 
	UCHIJAAAAAA	aquaculture{1.4.0} [Aquaculture] (Aquaculture-1.10.2-1.4.0.28.jar) 
	UCHIJAAAAAA	architecturecraft{1.7.1} [ArchitectureCraft] (ArchitectureCraft-1.7.1-mc1.10.2.jar) 
	UCHIJAAAAAA	Aroma1997CoreHelper{1.1.1.2} [Aroma1997Core|Helper] (Aroma1997Core-1.9.4-1.1.1.2.jar) 
	UCHIJAAAAAA	Aroma1997sDimension{1.0} [Aroma1997's Dimensional World] (Aroma1997s-Dimensional-World-1.9.4-1.2.0.7.jar) 
	UCHIJAAAAAA	arsmagica2{1.5.018} [Ars Magica 2] (ArsMagica2-1.10.2-1.5.0-18.jar) 
	UCHIJAAAAAA	AnimationAPI{1.2.4} [AnimationAPI] (ArsMagica2-1.10.2-1.5.0-18.jar) 
	UCHIJAAAAAA	Morphtool{1.0-12} [Morph-o-Tool] (Morph-o-Tool-1.0-12.jar) 
	UCHIJAAAAAA	Psi{r1.0-42} [Psi] (Psi-r1.0-42.jar) 
	UCHIJAAAAAA	Quark{r1.1-70} [Quark] (Quark-r1.1-70.jar) 
	UCHIJAAAAAA	AutoRegLib{1.0-2} [AutoRegLib] (AutoRegLib-1.0-2.jar) 
	UCHIJAAAAAA	crafttweakerjei{1.0.1} [CraftTweaker JEI Support] (CraftTweaker-1.10.2-3.0.25.jar) 
	UCHIJAAAAAA	MineTweaker3{3.0.25} [MineTweaker 3] (CraftTweaker-1.10.2-3.0.25.jar) 
	UCHIJAAAAAA	base{1.10.2-1.3.0} [B.A.S.E] (base-1.10.2-1.3.0.jar) 
	UCHIJAAAAAA	bedbugs{@VERSION@} [Bed Bugs] (BedBugs-1.10.2-1.1.6.jar) 
	UCHIJAAAAAA	BetterAchievements{0.3.3.10} [Better Achievements] (BetterAchievements-1.10.2-0.3.3.10.jar) 
	UCHIJAAAAAA	betterbuilderswands{0.11.1} [Better Builder's Wands] (BetterBuildersWands-1.10.2-0.11.1.220+f8232fe.jar) 
	UCHIJAAAAAA	betterthanbunnies{1.10.2-1.1.0} [BetterThanBunnies] (BetterThanBunnies-1.10.2-1.1.0.jar) 
	UCHIJAAAAAA	BiblioCraft{2.1.2} [BiblioCraft] (BiblioCraft[v2.2.2][MC1.10.2].jar) 
	UCHIJAAAAAA	guideapi{@VERSION@} [Guide-API] (Guide-API-1.10.2-2.0.3-46.jar) 
	UCHIJAAAAAA	BloodMagic{1.10.2-2.1.10-79} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.10.2-2.1.10-79.jar) 
	UCHIJAAAAAA	BloodArsenal{1.10.2-2.0.1-17} [Blood Arsenal] (BloodArsenal-1.10.2-2.0.1-17.jar) 
	UCHIJAAAAAA	bookshelf{1.4.4.347} [Bookshelf] (Bookshelf-1.10.2-1.4.4.347.jar) 
	UCHIJAAAAAA	CodeChickenLib{2.5.8.255} [CodeChicken Lib] (CodeChickenLib-1.10.2-2.5.8.255-universal.jar) 
	UCHIJAAAAAA	brandonscore{9.9.9} [Brandon's Core] (BrandonsCore-1.10.2-2.1.7.102-universal.jar) 
	UCHIJAAAAAA	ceramics{1.10.2-1.1.5} [Ceramics] (Ceramics-1.10.2-1.1.5.jar) 
	UCHIJAAAAAA	Chameleon{1.10.2-2.3.0} [Chameleon] (Chameleon-1.10.2-2.3.0.jar) 
	UCHIJAAAAAA	ChatFlow{ANT:VERSION} [ChatFlow] (ChatFlow-1.2-5.jar) 
	UCHIJAAAAAA	ChestTransporter{2.5.17} [Chest Transporter] (ChestTransporter-1.10.2-2.5.17.jar) 
	UCHIJAAAAAA	chisel{MC1.10.2-0.0.10.22} [Chisel] (Chisel-MC1.10.2-0.0.10.22.jar) 
	UCHIJAAAAAA	mcmultipart{1.4.0} [MCMultiPart] (MCMultiPart-1.4.0-universal.jar) 
	UCHIJAAAAAA	chiselsandbits{12.12} [Chisels & Bits] (chiselsandbits-12.12.jar) 
	UCHIJAAAAAA	shadowmc{3.6.1} [ShadowMC] (ShadowMC-1.10.2-3.6.1.jar) 
	UCHIJAAAAAA	clipboard{@VERSION@} [Clipboard] (Clipboard-1.10.2-1.1.1.jar) 
	UCHIJAAAAAA	clumps{1.0.5} [Clumps] (Clumps-1.0.5.jar) 
	UCHIJAAAAAA	CodeChickenCore{2.4.1.102} [CodeChicken Core] (CodeChickenCore-1.10.2-2.4.1.102-universal.jar) 
	UCHIJAAAAAA	compactsolars{1.10.2-5.0.9.332} [Compact Solar Arrays] (CompactSolars-1.10.2-5.0.9.332-universal.jar) 
	UCHIJAAAAAA	controlling{1.0.4} [Controlling] (Controlling-1.0.4.jar) 
	UCHIJAAAAAA	cookingforblockheads{4.2.43} [Cooking for Blockheads] (CookingForBlockheads_1.10.2-4.2.43.jar) 
	UCHIJAAAAAA	ctgui{1.0.0} [CT-GUI] (CraftTweaker-1.10.2-3.0.25.jar) 
	UCHIJAAAAAA	custommainmenu{2.0.5} [Custom Main Menu] (CustomMainMenu-MC1.10.2-2.0.5.jar) 
	UCHIJAAAAAA	cyberware{beta-0.2.10} [Cyberware] (cyberware-1.9.4-1.10.2-beta-0.2.10.jar) 
	UCHIJAAAAAA	darkutils{1.1.8.finalrc1} [Dark Utilities] (DarkUtilities-1.1.8.finalrc1.jar) 
	UCHIJAAAAAA	DeathCounter{MC1.9.0v2} [DeathCounter] (DeathCounter-1.10.2-1.0.0.jar) 
	UCHIJAAAAAA	deathquotes{1.2.0} [Death Quotes] (DeathQuotes-1.2.0-mc1.9to1.10.2-forge.jar) 
	UCHIJAAAAAA	PTRModelLib{1.0.0} [PTRModelLib] (Decocraft-2.4.2_1.10.2.jar) 
	UCHIJAAAAAA	props{2.4.2} [Decocraft] (Decocraft-2.4.2_1.10.2.jar) 
	UCHIJAAAAAA	forestry{5.2.17.376} [Forestry] (forestry_1.10.2-5.2.17.376.jar) 
	UCHIJAAAAAA	eleccore{1.7.421} [ElecCore] (ElecCore-1.10.2-1.7.421.jar) 
	UCHIJAAAAAA	EnderStorage{2.2.1.106} [EnderStorage] (EnderStorage-1.10.2-2.2.1.106-universal.jar) 
	UCHIJAAAAAA	OpenComputers{1.6.2.7} [OpenComputers] (OpenComputers-MC1.10.2-1.6.2.7.jar) 
	UCHIJAAAAAA	deepresonance{1.4.8} [DeepResonance] (deepresonance-1.1x-1.4.8.jar) 
	UCHIJAAAAAA	defaultoptions{6.1.5} [Default Options] (DefaultOptions_1.10.2-6.1.5.jar) 
	UCHIJAAAAAA	draconicevolution{2.0.12} [Draconic Evolution] (Draconic-Evolution-1.10.2-2.0.12.222-universal.jar) 
	UCHIJAAAAAA	StorageDrawers{1.10.2-3.7.7} [Storage Drawers] (StorageDrawers-1.10.2-3.7.7.jar) 
	UCHIJAAAAAA	bitdrawers{0.36} [Drawers & Bits] (DrawersBits-1.10.2-0.36.jar) 
	UCHIJAAAAAA	dsurround{3.4.5.0} [�3Dynamic Surroundings] (DynamicSurroundings-1.10.2-3.4.5.0.jar) 
	UCHIJAAAAAA	presets{3.4.5.0} [�3Presets!] (DynamicSurroundings-1.10.2-3.4.5.0.jar) 
	UCHIJAAAAAA	elevatorid{1.3.0} [Elevator Mod] (ElevatorMod[V.1.3.0][MC.1.10.2].jar) 
	UCHIJAAAAAA	embers{0.104} [Embers] (embers-0.104.jar) 
	UCHIJAAAAAA	eplus{4.1.0.113} [Enchanting Plus] (EnchantingPlus-1.10.2-4.1.0.115.jar) 
	UCHIJAAAAAA	endercore{1.10.2-0.4.1.66-beta} [EnderCore] (EnderCore-1.10.2-0.4.1.66-beta.jar) 
	UCHIJAAAAAA	EnderIO{1.10.2-3.1.186} [Ender IO] (EnderIO-1.10.2-3.1.186.jar) 
	UCHIJAAAAAA	engineersworkshop{1.0} [Engineers Workshop] (EngineersWorkshop-1.3.5-1.10.2.jar) 
	UCHIJAAAAAA	valkyrielib{1.10.2-0.10.6} [Valkyrie Lib] (valkyrielib-1.10.2-0.10.6.jar) 
	UCHIJAAAAAA	environmentaltech{1.10.2-0.10.6b} [Environmental Tech] (environmentaltech-1.10.2-0.10.6b.jar) 
	UCHIJAAAAAA	etlunar{1.10.2-0.10.6a} [ET Lunar] (etlunar-1.10.2-0.10.6a.jar) 
	UCHIJAAAAAA	galacticraftcore{4.0.0} [Galacticraft Core] (GalacticraftCore-1.10.2-4.0.0.100.jar) 
	UCHIJAAAAAA	galacticraftplanets{4.0.0} [Galacticraft Planets] (Galacticraft-Planets-1.10.2-4.0.0.100.jar) 
	UCHIJAAAAAA	extraplanets{0.4.3-Beta-Build} [Extra Planets] (ExtraPlanets-0.4.3-Beta-Build.jar) 
	UCHIJAAAAAA	extrautils2{1.0} [Extra Utilities 2] (extrautils2-1.10.2-1.4.1.jar) 
	UCHIJAAAAAA	zerocore{1.10.2-0.1.0.6} [Zero CORE] (zerocore-1.10.2-0.1.0.6.jar) 
	UCHIJAAAAAA	bigreactors{1.10.2-0.4.5.29} [Extreme Reactors] (ExtremeReactors-1.10.2-0.4.5.29.jar) 
	UCHIJAAAAAA	flatcoloredblocks{mc1.10-v4.4} [Flat Colored Blocks] (flatcoloredblocks-mc1.10-v4.4.jar) 
	UCHIJAAAAAA	sonarcore{3.2.7} [SonarCore] (SonarCore-1.10.2-3.2.7.jar) 
	UCHIJAAAAAA	fluxnetworks{1.2.3} [FluxNetworks] (FluxNetworks-1.10.2-1.2.3.jar) 
	UCHIJAAAAAA	foamfix{@VERSION@} [FoamFix] (foamfix-0.6.3-anarchy.jar) 
	UCHIJAAAAAA	forgelin{1.4.2} [Forgelin] (Forgelin-1.4.2.jar) 
	UCHIJAAAAAA	forgemultipartcbe{2.0.0.31} [Forge Multipart CBE] (ForgeMultipart-1.10.2-2.0.0.31-universal.jar) 
	UCHIJAAAAAA	microblockcbe{2.0.0.31} [Forge Microblocks] (ForgeMultipart-1.10.2-2.0.0.31-universal.jar) 
	UCHIJAAAAAA	minecraftmultipartcbe{2.0.0.31} [Minecraft Multipart Plugin] (ForgeMultipart-1.10.2-2.0.0.31-universal.jar) 
	UCHIJAAAAAA	ftbl{0.0.0} [FTBLib] (FTBLib-1.1x-3.6.5.jar) 
	UCHIJAAAAAA	ftbu{0.0.0} [FTBUtilities] (FTBUtilities-1.1x-3.6.5.jar) 
	UCHIJAAAAAA	funkylocomotion{1.0} [Funky Locomotion] (funky-locomotion-1.10.2-alpha-0.0.3.jar) 
	UCHIJAAAAAA	Funnels{1.1.2} [Funnels] (Funnels-1.10.2-1.1.2.jar) 
	UCHIJAAAAAA	gendustry{1.6.5.30} [GenDustry] (gendustry-1.6.5.30-mc1.10.2.jar) 
	UCHIJAAAAAA	gravestone{1.5.12} [Gravestone] (GraveStone Mod 1.5.12.jar) 
	UCHIJAAAAAA	gravisuite{3.0.1} [Gravitation Suite] (Gravitation Suite-3.0.1.jar) 
	UCHIJAAAAAA	ichunutil{6.3.2} [iChunUtil] (iChunUtil-1.10.2-6.3.2.jar) 
	UCHIJAAAAAA	gravitygun{6.0.1} [GravityGun] (GravityGun-1.10.2-6.0.1.jar) 
	UCHIJAAAAAA	hammercore{1.6.9} [Hammer Core] (HammerCore_1.10.2-1.6.9.jar) 
	UCHIJAAAAAA	helpfixer{1.0} [HelpFixer] (HelpFixer-1.10.2-1.4.15.jar) 
	UCHIJAAAAAA	hopperducts{1.4.8} [Hopper Ducts] (hopperducts-mc1.9.4-1.4.8.jar) 
	UCHIJAAAAAA	mantle{1.10.2-1.1.5.205} [Mantle] (Mantle-1.10.2-1.1.5.jar) 
	UCHIJAAAAAA	tconstruct{1.10.2-2.6.5.10} [Tinkers' Construct] (TConstruct-1.10.2-2.6.5.jar) 
	UCHIJAAAAAA	theoneprobe{1.4.7} [The One Probe] (theoneprobe-1.1x-1.4.7.jar) 
	UCHIJAAAAAA	immersiveengineering{0.10-58} [Immersive Engineering] (ImmersiveEngineering-0.10-58.jar) 
	UCHIJAAAAAA	LunatriusCore{1.1.2.38} [LunatriusCore] (LunatriusCore-1.10.2-1.1.2.38-universal.jar) 
	UCHIJAAAAAA	InGameInfoXML{2.8.1.89} [InGame Info XML] (InGameInfoXML-1.10.2-2.8.1.89-universal.jar) 
	UCHIJAAAAAA	intwheel{1.2.6} [Interaction Wheel] (intwheel-1.1x-1.2.6.jar) 
	UCHIJAAAAAA	inventorytweaks{1.61-58-a1fd884} [Inventory Tweaks] (InventoryTweaks-1.61-58.jar) 
	UCHIJAAAAAA	ironbackpacks{1.10.2-2.2.26} [Iron Backpacks] (IronBackpacks-1.10.2-2.2.26.jar) 
	UCHIJAAAAAA	ironchest{1.10.2-7.0.15.804} [Iron Chest] (ironchest-1.10.2-7.0.15.804.jar) 
	UCHIJAAAAAA	ItsRainingFood{1.0.2} [It's Raining Food] (itsrainingfood-1.10.2-0.1.0.jar) 
	UCHIJAAAAAA	jeibees{0.9.0.5} [JEI Bees] (jeibees-0.9.0.5-mc1.10.2.jar) 
	UCHIJAAAAAA	journeymap{1.10.2-5.4.6} [JourneyMap] (journeymap-1.10.2-5.4.6.jar) 
	UCHIJAAAAAA	jeresources{0.5.8.98} [Just Enough Resources] (JustEnoughResources-1.10.2-0.5.8.98.jar) 
	UCHIJAAAAAA	kleeslabs{3.3.5} [KleeSlabs] (KleeSlabs_1.10.2-3.3.5.jar) 
	UCHIJAAAAAA	launchgui{@VERSION@} [LaunchGUI] (LaunchGui-1.9.4-2.0.3-23-client.jar) 
	UCHIJAAAAAA	boots{1.0} [Long Fall Boots] (LongFallBoots - Hard Mode.jar) 
	UCHIJAAAAAA	malisiscore{1.10.2-4.4.0} [MalisisCore] (malisiscore-1.10.2-4.4.0.jar) 
	UCHIJAAAAAA	malisisadvert{1.10.2-4.2.0} [Malisis Advert] (malisisadvert-1.10.2-4.2.0.jar) 
	UCHIJAAAAAA	malisisdoors{1.10.2-5.2.0} [MalisisDoors] (malisisdoors-1.10.2-5.2.0.jar) 
	UCHIJAAAAAA	malisisswitches{1.10.2-3.2.0} [Malisis Switches] (malisisswitches-1.10.2-3.2.0.jar) 
	UCHIJAAAAAA	Mekanism{9.2.2} [Mekanism] (Mekanism-1.10.2-9.2.2.301.jar) 
	UCHIJAAAAAA	MekanismGenerators{9.2.2} [MekanismGenerators] (MekanismGenerators-1.10.2-9.2.2.301.jar) 
	UCHIJAAAAAA	MekanismTools{9.2.2} [MekanismTools] (MekanismTools-1.10.2-9.2.2.301.jar) 
	UCHIJAAAAAA	mercurius{1.0.4} [Mercurius] (Mercurius-1.10.2-1.0.4.jar) 
	UCHIJAAAAAA	modularforcefieldsystem{3.0.1} [Modular ForceField System] (MFFS-1.10.2-3.0.1.0_1.10_d9fd06a.jar) 
	UCHIJAAAAAA	testdummy{1.11} [MmmMmmMmmMmm] (MmmMmmMmmMmm-1.10.2-1.13.jar) 
	UCHIJAAAAAA	mob_grinding_utils{0.1.8.21} [mob_grinding_utils] (MobGrindingUtils-0.1.8.21.jar) 
	UCHIJAAAAAA	ModNameTooltip{1.6.0} [Mod Name Tooltip] (modnametooltip_1.10.2-1.6.0.jar) 
	UCHIJAAAAAA	mtlib{@VERSION@} [MTLib] (MTLib-1.0.3.jar) 
	UCHIJAAAAAA	modtweaker{2.0.12} [Mod Tweaker] (ModTweaker2-2.0.12.jar) 
	UCHIJAAAAAA	numina{1.4.0} [numina] (ModularPowersuits-1.10.2-1.2.118.jar) 
	UCHIJAAAAAA	powersuits{1.10.2-1.2.118} [MachineMuse's Modular Powersuits] (ModularPowersuits-1.10.2-1.2.118.jar) 
	UCHIJAAAAAA	railcraft{10.1.2} [Railcraft] (Railcraft_1.10.2-10.1.2.jar) 
	UCHIJAAAAAA	morebees{1.10.2-1.3.2} [More Bees] (morebees-1.10.2-1.3.2.jar) 
	UCHIJAAAAAA	Morpheus{1.10.2-3.1.13} [Morpheus] (Morpheus-1.10.2-3.1.13.jar) 
	UCHIJAAAAAA	mousetweaks{2.8} [Mouse Tweaks] (MouseTweaks-2.8-mc1.10.2.jar) 
	UCHIJAAAAAA	mputils{1.2.8} [ModPack Utilities] (mputils-1.10.2-1.2.8.jar) 
	UCHIJAAAAAA	mpbasic{1.1.3} [ModPack Basic Tools] (mpbasic-1.10.2-1.1.5.jar) 
	UCHIJAAAAAA	MrTJPCoreMod{2.0.0.17} [MrTJPCore] (MrTJPCore-1.10.2-2.0.0.17-universal.jar) 
	UCHIJAAAAAA	MTR{1.10.2-2.1.1} [Minecraft Transit Railway] (MTR-1.10.2-2.1.1.jar) 
	UCHIJAAAAAA	shetiphiancore{3.3.9} [ShetiPhian-Core] (shetiphiancore-1.10.0-3.3.9.jar) 
	UCHIJAAAAAA	multistorage{1.2.18} [Multi-Storage] (multistorage-1.10.0-1.2.18.jar) 
	UCHIJAAAAAA	natura{1.10.2-4.1.0.63} [Natura] (natura-1.10.2-4.1.0.63.jar) 
	UCHIJAAAAAA	netherportalfix{1.0} [Nether Portal Fix] (netherportalfix-mc1.9.4-4.0.1.jar) 
	UCHIJAAAAAA	notenoughwands{1.5.5} [Not Enough Wands] (notenoughwands-1.1x-1.5.5.jar) 
	UCHIJAAAAAA	omlib{3.0.0-67} [OMLib] (omlib-1.10.2-3.0.0-94.jar) 
	UCHIJAAAAAA	openglider{@VERSION@} [Open Glider] (OpenGlider-1.10.2-0.2.3.jar) 
	UCHIJAAAAAA	openmodularturrets{1.10.2-3.0.0-153} [Open Modular Turrets] (openmodularturrets-1.10.2-3.0.0-153.jar) 
	UCHIJAAAAAA	p455w0rdslib{1.0.13} [p455w0rd's Library] (p455w0rdslib-1.10.2-1.0.13.jar) 
	UCHIJAAAAAA	harvestcraft{1.9.4-1.10.2g} [Pam's HarvestCraft] (Pam's HarvestCraft 1.9.4-1.10.2h.jar) 
	UCHIJAAAAAA	projectred-core{4.8.2.41} [ProjectRed Core] (ProjectRed-1.10.2-4.8.2.41-Base.jar) 
	UCHIJAAAAAA	projectred-integration{4.8.2.41} [ProjectRed integration] (ProjectRed-1.10.2-4.8.2.41-integration.jar) 
	UCHIJAAAAAA	projectred-transmission{4.8.2.41} [ProjectRed Transmission] (ProjectRed-1.10.2-4.8.2.41-integration.jar) 
	UCHIJAAAAAA	projectred-illumination{4.8.2.41} [ProjectRed Illumination] (ProjectRed-1.10.2-4.8.2.41-lighting.jar) 
	UCHIJAAAAAA	projectred-expansion{4.8.2.41} [ProjectRed Expansion] (ProjectRed-1.10.2-4.8.2.41-mechanical.jar) 
	UCHIJAAAAAA	projectred-transportation{4.8.2.41} [ProjectRed Transportation] (ProjectRed-1.10.2-4.8.2.41-mechanical.jar) 
	UCHIJAAAAAA	projectred-exploration{4.8.2.41} [ProjectRed Exploration] (ProjectRed-1.10.2-4.8.2.41-world.jar) 
	UCHIJAAAAAA	reborncore{2.13.6.142} [RebornCore] (RebornCore-1.10.2-2.13.6.142-universal.jar) 
	UCHIJAAAAAA	quantumstorage{3.3.4} [QuantumStorage] (QuantumStorage-1.10.2-3.3.4.jar) 
	UCHIJAAAAAA	rangedpumps{0.3} [Ranged Pumps] (rangedpumps-0.3.jar) 
	UCHIJAAAAAA	reauth{3.4.1} [ReAuth] (ReAuth-1.10-3.4.1.jar) 
	UCHIJAAAAAA	reborncore-mcmultipart{2.13.6.142} [reborncore-MCMultiPart] (RebornCore-1.10.2-2.13.6.142-universal.jar) 
	UCHIJAAAAAA	thermalfoundation{2.0.5} [Thermal Foundation] (ThermalFoundation-1.10.2-2.0.5.81-universal.jar) 
	UCHIJAAAAAA	redstonearsenal{2.0.3} [Redstone Arsenal] (RedstoneArsenal-1.10.2-2.0.3.29-universal.jar) 
	UCHIJAAAAAA	refinedrelocation{@VERSION@} [Refined Relocation] (RefinedRelocation-mc1.10.2-2.1.23.jar) 
	UCHIJAAAAAA	refinedstorage{1.2.25} [Refined Storage] (refinedstorage-1.2.25.jar) 
	UCHIJAAAAAA	xreliquary{1.10.2-1.3.3.664} [Reliquary] (Reliquary-1.10.2-1.3.3.664.jar) 
	UCHIJAAAAAA	ResourceLoader{1.5.1} [Resource Loader] (ResourceLoader-MC1.9.4-1.5.1.jar) 
	UCHIJAAAAAA	rftools{6.01} [RFTools] (rftools-1.1x-6.01.jar) 
	UCHIJAAAAAA	rftoolscontrol{1.6.8} [RFTools Control] (rftoolsctrl-1.1x-1.6.8.jar) 
	UCHIJAAAAAA	rftoolsdim{4.55} [RFTools Dimensions] (rftoolsdim-1.1x-4.55.jar) 
	UCHIJAAAAAA	roots{0.208} [Roots] (roots-0.208.jar) 
	UCHIJAAAAAA	ropebridge{2.0.5} [Rope Bridge] (ropebridge-1.10.2-2.0.5.jar) 
	UCHIJAAAAAA	sgcraft{1.13.2} [SG Craft] (SGCraft-1.13.2-mc1.10.2.jar) 
	UCHIJAAAAAA	Signals{1.0} [Signals] (Signals-1.10.2-0.5.1-universal.jar) 
	UCHIJAAAAAA	simplevoidworld{1.1.0.4} [SimpleVoidWorld] (Simple-Void-World-1.10.2-1.1.0.4-universal.jar) 
	UCHIJAAAAAA	simpleretrogen{4.0.1+8} [Simple Retrogen] (simpleretrogen-1.10-4.0.1+8.jar) 
	UCHIJAAAAAA	simplyjetpacks{2.1.1.47} [Simply Jetpacks 2] (SimplyJetpacks2-1.10.2-2.1.1.47.jar) 
	UCHIJAAAAAA	SleepingBag{1.4.0} [Sleeping Bag] (SleepingBag-1.10.2-1.4.0.jar) 
	UCHIJAAAAAA	snad{1.10.2-1.6.09.20a} [Snad] (Snad-1.10.2-1.6.09.20a.jar) 
	UCHIJAAAAAA	solarfluxreborn{2.17r} [Solar Flux Reborn] (SolarFluxReborn_1.10.2-2.17r.jar) 
	UCHIJAAAAAA	stevescarts{2.2.0.86} [Steve's Carts 2] (StevesCarts-1.10.2-2.2.0.86.jar) 
	UCHIJAAAAAA	storagedrawersextra{1.10-1.1.0} [Storage Drawers Extras] (StorageDrawersExtras-1.10-1.1.0.jar) 
	UCHIJAAAAAA	rscircuits{1.0.4} [Super Circuit Maker] (SuperCircuitMaker-1.0.4.jar) 
	UCHIJAAAAAA	supercraftingframe{1.2.1.88} [Super Crafting Frame] (supercraftingframe-1.10.2-1.2.1.88.jar) 
	UCHIJAAAAAA	techreborn{2.1.13.219} [TechReborn] (TechReborn-1.10.2-2.1.13.219-universal.jar) 
	UCHIJAAAAAA	thermalexpansion{5.0.4} [Thermal Expansion] (ThermalExpansion-1.10.2-5.0.4.90-universal.jar) 
	UCHIJAAAAAA	tinkertoolleveling{1.10.2-1.0.1.DEV.f5def58} [Tinkers Tool Leveling] (TinkerToolLeveling-1.10.2-1.0.1.jar) 
	UCHIJAAAAAA	topaddons{1.10.2-0.13.0} [TOP Addons] (topaddons-1.10.2-0.13.0.jar) 
	UCHIJAAAAAA	torchmaster{1.0} [TorchMaster] (torchmaster_1.10.2-1.2.3.27.jar) 
	UCHIJAAAAAA	Translocator{2.1.4.55} [Translocator] (Translocators-1.10.2-2.1.4.55-universal.jar) 
	UCHIJAAAAAA	travelhut{1.0.8} [Travel Huts] (travelhut-1.0.8.jar) 
	UCHIJAAAAAA	usefulnullifiers{1.3.4} [Useful Nullifiers] (usefulnullifiers-1.3.4.jar) 
	UCHIJAAAAAA	vending{1.4.3} [Vending] (vending-1.10.2-1.4.3.jar) 
	UCHIJAAAAAA	wanionlib{1.10.2-1.3} [WanionLib] (WanionLib-1.10.2-1.3.jar) 
	UCHIJAAAAAA	wcg{1.0.0} [Wireless Crafting Grid] (WirelessCraftingGrid-1.10.2-1.0.0.jar) 
	UCHIJAAAAAA	wct{0.1.31} [Wireless Crafting Terminal 2] (WirelessCraftingTerminal-1.10.2-0.1.31.jar) 
	UCHIJAAAAAA	Woot{1.10.2-1.4.0} [Woot] (woot-1.10.2-1.4.0.jar) 
	UCHIJAAAAAA	wrcbe{2.0.0} [WirelessRedstone-CBE] (WR-CBE-1.10.2-2.0.0.18-universal.jar) 
	UCHIJAAAAAA	xnet{1.3.2} [XNet] (xnet-1.3.2.jar) 
	UCHIJAAAAAA	yabba{1.10.2-0.5.1} [YABBA] (YABBA-1.10.2-0.5.1.jar) 
	UCHIJAAAAAA	unidict{1.10.2-2.8} [UniDict] (UniDict-1.10.2-2.8.jar) 
	Loaded coremods (and transformers): 
CoreMod (Aroma1997Core-1.9.4-1.1.1.2.jar)
  
Do not report to Forge! Remove FoamFixAPI (or replace with FoamFixAPI-Lawful) and try again. (foamfix-0.6.3-anarchy.jar)
  pl.asie.foamfix.coremod.FoamFixTransformer
SFRCore (SolarFluxReborn_1.10.2-2.17r.jar)
  com.mrdimka.solarfluxreborn.core.SFRSerializedTransformer
MalisisCorePlugin (malisiscore-1.10.2-4.4.0.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
  net.malisis.core.util.clientnotif.ClientNotifTransformer
EnderCorePlugin (EnderCore-1.10.2-0.4.1.66-beta.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
ForgelinPlugin (Forgelin-1.4.2.jar)
  
MalisisSwitchesPlugin (malisisswitches-1.10.2-3.2.0.jar)
  net.malisis.switches.asm.MalisisSwitchesTransformer
MicdoodlePlugin (MicdoodleCore-1.10.2-4.0.0.100.jar)
  micdoodle8.mods.miccore.MicdoodleTransformer
ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.9.jar)
  shetiphian.core.asm.ClassTransformer
Default Options (DefaultOptions_1.10.2-6.1.5.jar)
  net.blay09.mods.defaultoptions.coremod.DefaultOptionsClassTransformer
NEICorePlugin (NotEnoughItems-1.10.2-2.1.3.220-universal.jar)
  codechicken.nei.asm.NEITransformer
RebornCoreASM (RebornCore-1.10.2-2.13.6.142-universal.jar)
  reborncore.mixin.transformer.MixinTransformer
FMLPlugin (InventoryTweaks-1.61-58.jar)
  invtweaks.forge.asm.ContainerTransformer
CorePlugin (MrTJPCore-1.10.2-2.0.0.17-universal.jar)
  
BookshelfLoadingPlugin (Bookshelf-1.10.2-1.4.4.347.jar)
  net.darkhax.bookshelf.asm.BookshelfTransformerManager
IC2core (industrialcraft-2-2.6.210-ex110.jar)
  
DynamicSurroundingsCore (DynamicSurroundings-1.10.2-3.4.5.0.jar)
  org.blockartistry.DynSurround.asm.Transformer
LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  lumien.resourceloader.asm.ClassTransformer
Brandon's Core (BrandonsCore-1.10.2-2.1.7.102-universal.jar)
  com.brandon3055.brandonscore.asm.ClassTransformer
ChiselCorePlugin (Chisel-MC1.10.2-0.0.10.22.jar)
  team.chisel.common.asm.ChiselTransformer
CCLCorePlugin (CodeChickenLib-1.10.2-2.5.8.255-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.CCL_ASMTransformer
TransformerLoader (OpenComputers-MC1.10.2-1.6.2.7.jar)
  li.cil.oc.common.asm.ClassTransformer
HammerCoreCore (HammerCore_1.10.2-1.6.9.jar)
  com.mrdimka.hammercore.asm.HammerCoreTransformer
LoadingPlugin (Quark-r1.1-70.jar)
  vazkii.quark.base.asm.ClassTransformer
AppEngCore (appliedenergistics2-rv4-alpha-11.jar)
  appeng.coremod.transformer.ASMIntegration
ArsMagica2-Preloader (ArsMagica2-1.10.2-1.5.0-18.jar)
  am2.asm.Transformer
CoFH Loading Plugin (CoFHCore-1.10.2-4.1.1.156-universal.jar)
  cofh.asm.CoFHClassTransformer
  cofh.asm.repack.codechicken.lib.asm.ClassHierarchyManager
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 378.78' Renderer: 'GeForce GTX 780/PCIe/SSE2'
	AE2 Version: alpha rv4-alpha-11 for Forge 12.18.3.2185
	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)
		- chiselsandbitsIntegration (Enabled/Not Forced)
		- theoneprobeIntegration (Enabled/Not Forced)

	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaDecorative (Enabled/Not Forced)
		- NaturaTools (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)

	NotEnoughItems Invalid Fingerprint Reports: 
	List of loaded APIs: 
		* actuallyadditionsapi (29) from ActuallyAdditions-1.10.2-r105.jar
		* AgriCraftAPI (1.0) from agricraft-2.0.0-0.10.0-a17.jar
		* appliedenergistics2|API (rv4) from appliedenergistics2-rv4-alpha-11.jar
		* Base|API (1.0.0) from base-1.10.2-1.3.0.jar
		* Baubles|API (1.3.1.3) from WirelessCraftingTerminal-1.10.2-0.1.31.jar
		* BetterAchievements|API (0.3.3.10) from BetterAchievements-1.10.2-0.3.3.10.jar
		* bigreactors|API (4.0.1) from ExtremeReactors-1.10.2-0.4.5.29.jar
		* BloodMagic|API (@VERSION@) from BloodMagic-1.10.2-2.1.10-79.jar
		* BotaniaAPI (85) from Botania r1.9-341.jar
		* chesttransporter|api (1) from ChestTransporter-1.10.2-2.5.17.jar
		* ChiselAPI (0.0.1) from Chisel-MC1.10.2-0.0.10.22.jar
		* ChiselAPI|Carving (0.0.1) from Chisel-MC1.10.2-0.0.10.22.jar
		* ChiselsAndBitsAPI (12.10.0) from chiselsandbits-12.12.jar
		* CoFHAPI (1.8.9R1.2.0B1) from refinedstorage-1.2.25.jar
		* cofhapi (1.5.0) from SimplyJetpacks2-1.10.2-2.1.1.47.jar
		* cofhapi|block (1.6.0) from zerocore-1.10.2-0.1.0.6.jar
		* CoFHAPI|core (1.8.9R1.2.0B1) from valkyrielib-1.10.2-0.10.6.jar
		* cofhapi|core (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* CoFHAPI|energy (1.8.9R1.2.0B1) from Mekanism-1.10.2-9.2.2.301.jar
		* cofhapi|energy (1.5.0) from SimplyJetpacks2-1.10.2-2.1.1.47.jar
		* CoFHAPI|inventory (1.8.9R1.2.0B1) from valkyrielib-1.10.2-0.10.6.jar
		* CoFHAPI|item (1.8.9R1.2.0B1) from valkyrielib-1.10.2-0.10.6.jar
		* cofhapi|item (1.6.0) from zerocore-1.10.2-0.1.0.6.jar
		* cofhapi|tileentity (1.6.0) from zerocore-1.10.2-0.1.0.6.jar
		* cofhapi|util (1.6.0) from zerocore-1.10.2-0.1.0.6.jar
		* cofhlib (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|audio (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|gui (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|gui|container (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|gui|element (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|gui|element|listbox (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|gui|slot (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|inventory (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|util (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|util|helpers (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|world (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* cofhlib|world|feature (1.6.0) from CoFHCore-1.10.2-4.1.1.156-universal.jar
		* compatlayer (0.2.9) from compatlayer-1.10-0.2.9.jar
		* ComputerCraft|API (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|FileSystem (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Lua (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Media (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Peripheral (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Permissions (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Redstone (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* ComputerCraft|API|Turtle (1.80pr0) from zerocore-1.10.2-0.1.0.6.jar
		* CSLib|API (0.3.0) from Decocraft-2.4.2_1.10.2.jar
		* DR-API (1.0.4-Beta) from deepresonance-1.1x-1.4.8.jar
		* DraconicEvolution|API (1.3) from Draconic-Evolution-1.10.2-2.0.12.222-universal.jar
		* ElecCoreAPI (1.0) from ElecCore-1.10.2-1.7.421.jar
		* ElecCore|Abilities (#API_VER#) from ElecCore-1.10.2-1.7.421.jar
		* EnderIOAPI (0.0.2) from EnderIO-1.10.2-3.1.186.jar
		* EnderIOAPI|Redstone (0.0.2) from EnderIO-1.10.2-3.1.186.jar
		* EnderIOAPI|Teleport (0.0.2) from EnderIO-1.10.2-3.1.186.jar
		* EnderIOAPI|Tools (0.0.2) from EnderIO-1.10.2-3.1.186.jar
		* environmentaltech-api (1.10.2-0.10.6b) from environmentaltech-1.10.2-0.10.6b.jar
		* fluxapi (1.0) from FluxNetworks-1.10.2-1.2.3.jar
		* ForestryAPI|apiculture (5.0.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|arboriculture (4.2.1) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|circuits (3.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|climate (2.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|core (5.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|farming (2.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|food (1.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|fuels (3.0.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|genetics (4.7.1) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|greenhouse (5.2.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|hives (4.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|lepidopterology (1.3.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|mail (3.1.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|multiblock (3.0.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|recipes (5.4.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|storage (5.0.0) from forestry_1.10.2-5.2.17.376.jar
		* ForestryAPI|world (2.1.0) from forestry_1.10.2-5.2.17.376.jar
		* funkylocomotion_api (2.0) from funky-locomotion-1.10.2-alpha-0.0.3.jar
		* Galacticraft API (1.0) from GalacticraftCore-1.10.2-4.0.0.100.jar
		* gendustryAPI (2.3.0) from gendustry-1.6.5.30-mc1.10.2.jar
		* Guide-API|API (1.10.2-2.0.3-46) from Guide-API-1.10.2-2.0.3-46.jar
		* iChunUtil API (1.2.0) from iChunUtil-1.10.2-6.3.2.jar
		* ImmersiveEngineering|API (1.0) from ImmersiveEngineering-0.10-58.jar
		* ImmersiveEngineering|ImmersiveFluxAPI (1.0) from ImmersiveEngineering-0.10-58.jar
		* intwheel_api (1.2.4) from intwheel-1.1x-1.2.6.jar
		* IronBackpacks|API (0.5) from IronBackpacks-1.10.2-2.2.26.jar
		* jeresources|API (0.5.8.98) from JustEnoughResources-1.10.2-0.5.8.98.jar
		* journeymap|client-api (1.3) from journeymap-1.10.2-5.4.6.jar
		* journeymap|client-api-display (1.3) from journeymap-1.10.2-5.4.6.jar
		* journeymap|client-api-event (1.3) from journeymap-1.10.2-5.4.6.jar
		* journeymap|client-api-model (1.3) from journeymap-1.10.2-5.4.6.jar
		* journeymap|client-api-util (1.3) from journeymap-1.10.2-5.4.6.jar
		* JustEnoughItemsAPI (4.10.1) from jei_1.10.2-3.14.7.419.jar
		* mcjtylib_ng (2.3.11) from mcjtylib-1.1x-2.3.11.jar
		* MekanismAPI|core (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|energy (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|gas (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|infuse (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|laser (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|reactor (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|recipe (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|transmitter (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MekanismAPI|util (9.0.0) from Mekanism-1.10.2-9.2.2.301.jar
		* MouseTweaks|API (1.0) from MouseTweaks-2.8-mc1.10.2.jar
		* Open Glider|API (0.1) from OpenGlider-1.10.2-0.2.3.jar
		* OpenComputersAPI|Component (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Core (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Driver (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Driver|Item (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Event (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|FileSystem (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Internal (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Machine (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Manual (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Network (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* OpenComputersAPI|Prefab (6.0.0-alpha) from OpenComputers-MC1.10.2-1.6.2.7.jar
		* ProjectEAPI (1.9.4-1.0.0) from p455w0rdslib-1.10.2-1.0.13.jar
		* PsiAPI (2) from Psi-r1.0-42.jar
		* railcraft:API|core (3.1.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|bore (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|carts (2.1.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|charge (3.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|crafting (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|events (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|fuel (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|helpers (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|items (2.1.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|locomotive (2.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|signals (4.0.0) from Railcraft_1.10.2-10.1.2.jar
		* RailcraftAPI|tracks (4.0.0) from Railcraft_1.10.2-10.1.2.jar
		* reborncoreAPI (2.13.6.142) from RebornCore-1.10.2-2.13.6.142-universal.jar
		* reborncoreAPI|Fuel (2.13.6.142) from RebornCore-1.10.2-2.13.6.142-universal.jar
		* reborncoreAPI|Power (2.13.6.142) from RebornCore-1.10.2-2.13.6.142-universal.jar
		* reborncoreAPI|Recipe (2.13.6.142) from RebornCore-1.10.2-2.13.6.142-universal.jar
		* reborncoreAPI|Tile (2.13.6.142) from RebornCore-1.10.2-2.13.6.142-universal.jar
		* RefinedRelocationAPI (2.0.0) from RefinedRelocation-mc1.10.2-2.1.23.jar
		* sonarapi (1.0.1) from SonarCore-1.10.2-3.2.7.jar
		* stevescartsAPI (2.2.0.86) from StevesCarts-1.10.2-2.2.0.86.jar
		* stevescartsAPI|FARMS (2.2.0.86) from StevesCarts-1.10.2-2.2.0.86.jar
		* StorageDrawersAPI (1.7.10-1.2.0) from refinedstorage-1.2.25.jar
		* StorageDrawersAPI|config (1.10.2-1.3.0) from StorageDrawers-1.10.2-3.7.7.jar
		* StorageDrawersAPI|event (1.10.2-1.3.0) from StorageDrawers-1.10.2-3.7.7.jar
		* StorageDrawersAPI|inventory (1.7.10-1.2.0) from refinedstorage-1.2.25.jar
		* StorageDrawersAPI|pack (1.10.2-1.3.0) from StorageDrawers-1.10.2-3.7.7.jar
		* StorageDrawersAPI|registry (1.7.10-1.2.0) from refinedstorage-1.2.25.jar
		* StorageDrawersAPI|render (1.7.10-1.2.0) from refinedstorage-1.2.25.jar
		* StorageDrawersAPI|storage (1.10.2-1.3.0) from StorageDrawers-1.10.2-3.7.7.jar
		* StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from refinedstorage-1.2.25.jar
		* techrebornAPI (2.1.13.219) from TechReborn-1.10.2-2.1.13.219-universal.jar
		* theoneprobe_api (1.4.4) from theoneprobe-1.1x-1.4.7.jar
		* valkyrielib.api (1.10.2-0.10.6) from valkyrielib-1.10.2-0.10.6.jar
		* wct|API (1.10.2-rv4-1.0.0b) from WirelessCraftingTerminal-1.10.2-0.1.31.jar
		* zerocore|API|multiblock (1.10.2-0.0.2) from zerocore-1.10.2-0.1.0.6.jar
		* zerocore|API|multiblock|rectangular (1.10.2-0.0.2) from zerocore-1.10.2-0.1.0.6.jar
		* zerocore|API|multiblock|tier (1.10.2-0.0.2) from zerocore-1.10.2-0.1.0.6.jar
		* zerocore|API|multiblock|validation (1.10.2-0.0.2) from zerocore-1.10.2-0.1.0.6.jar
	CodeChickenLib Invalid Fingerprint Reports: 
	CodeChickenCore Invalid Fingerprint Reports: 
	EnderStorage Invalid Fingerprint Reports: 
	EnderIO: Found the following problem(s) with your installation (That does NOT mean that Ender IO caused the crash or was involved in it in any way. We add this information to help finding common problems, not as an invitation to post any crash you encounter to Ender IO's issue tracker. Always check the stack trace above to see which mod is most likely failing.):
                  * Optifine is installed. This is NOT supported.
                  * An unsupportted RF API is installed (1.7.10R1.3.1 from (guessing) jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/CoFHAPIProps.class).
                    Ender IO needs at least 1.4.0 and will NOT work with older versions.
                 This may (look up the meaning of 'may' in the dictionary if you're not sure what it means) have caused the error. Try reproducing the crash WITHOUT this/these mod(s) before reporting it.
	Detailed RF API diagnostics:
                  * RF API class 'EnergyStorage' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/EnergyStorage.class
                  * RF API class 'IEnergyConnection' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyConnection.class
                  * RF API class 'IEnergyContainerItem' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyContainerItem.class
                  * RF API class 'IEnergyHandler' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyHandler.class
                  * RF API class 'IEnergyProvider' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyProvider.class
                  * RF API class 'IEnergyReceiver' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyReceiver.class
                  * RF API class 'IEnergyStorage' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/IEnergyStorage.class
                  * RF API class 'ItemEnergyContainer' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/ItemEnergyContainer.class
                  * RF API class 'TileEnergyHandler' is loaded from: jar:file:/C:/Users/bvand/AppData/Roaming/.technic/modpacks/dbmpack/mods/BrandonsCore-1.10.2-2.1.7.102-universal.jar!/cofh/api/energy/TileEnergyHandler.class
	Detailed Tesla API diagnostics:
                  * Tesla API class 'Tesla' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.Tesla)
                  * Tesla API class 'TeslaCapabilities' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.capability.TeslaCapabilities)
                  * Tesla API class 'ITeslaConsumer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaConsumer)
                  * Tesla API class 'ITeslaHolder' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaHolder)
                  * Tesla API class 'ITeslaProducer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaProducer)
                  * Tesla API class 'BaseTeslaContainer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.BaseTeslaContainer)
                  * Tesla API class 'BaseTeslaContainerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.BaseTeslaContainerProvider)
                  * Tesla API class 'InfiniteTeslaConsumer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaConsumer)
                  * Tesla API class 'InfiniteTeslaConsumerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaConsumerProvider)
                  * Tesla API class 'InfiniteTeslaProducer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaProducer)
                  * Tesla API class 'InfiniteTeslaProducerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaProducerProvider)

	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
	!!!You are looking at the diagnostics information, not at the crash. Scroll up!!!
	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

	RebornCore: 
		Plugin Engine: 0
		RebornCore Version: 2.13.6.142
		Mixin Status: 1
		Runtime Debofucsation 1
		RenderEngine: 1
	Translocator Invalid Fingerprint Reports: 
	AE2 Integration: IC2:ON, RC:OFF, RF:OFF, RFItem:OFF, MFR:OFF, Waila:OFF, InvTweaks:OFF, JEI:ON, Mekanism:OFF, OpenComputers:OFF, THE_ONE_PROBE:ON
	Launched Version: 1.10.2-forge1.10.2-12.18.3.2297
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 780/PCIe/SSE2 GL version 4.5.0 NVIDIA 378.78, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 12x Intel(R) Core(TM) i7-3930K CPU @ 3.20GHz
	OptiFine Version: OptiFine_1.10.2_HD_U_D8
	Render Distance Chunks: 8
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.5.0 NVIDIA 378.78
	OpenGlRenderer: GeForce GTX 780/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 12

**Comment by radfast** [comment]
Was this a one time only thing, or it's now happening every time in that world?

**Comment by radfast** [comment]
The reported crash shold be fixed in next build 103.

The 5 minutes wait on teleport thing we are also looking at, if you can provide the info requested that will help

**Comment by radfast** [comment]
Build 104 now available with this fix.

**Comment by radfast** [comment]
Please test build 106 and let us know if that "5 minutes wait on teleport" issue is still happening or if it's improved...

**Comment by marcus8448** [comment]
I'm getting this error on my server too...
**Modlist:
Galacticraft Build 181
Forge at 1.12.2-14.23.4.2705**
chameleon
mousetweaks
inventorytweaks
foamfixcore
hats
cfm
waila
foamfix
jei
galacticraftplanets
ichunutil
micdoodlecore
durabilityshow
wts (whats that slot)
torohealthmod
xaerominima
morph
storagedrawers

No crash but haven't gotten past the screen yet, tried logging out and back in but that just spawns the player (motionless) at y=800. Tried to `/kill` and `/tp` but chunks won't load/I'm stuck.
Server debug log: https://gist.github.com/marcus8448/33298d3d65a464570f05ebc1d1757d6c
In the end I Alt+F4 ed the game then stopped the server.
If you want I can do this on another issue.
